src/views/AccountDetail.vue

```diff
@@ -30,7 +30,7 @@
 
       </Card>
 
-      <Card v-if="1||showMosaics" class="card-f card-adaptive" :loading="loading">
```

Comment:
i think `showMosaics ` is valild because it existing in `computed` , only `1||` have to remove.